mne/export/_export.py

```diff
@@ -10,7 +10,7 @@
 
 
 @verbose
-def export_raw(fname, raw, fmt='auto', verbose=None):
+def export_raw(fname, raw, fmt='auto', physical_range='auto', verbose=None):
```

Comment:
Physical range is specific to EDF/BDF, but having a dedicated parameter here hides this fact. I'd catch it as a `**kwarg` and use it only in the EDF/BDF part of `_export_raw()`.

Reply:
Done